x-pack/plugins/index_management/public/application/services/api.ts

```diff
@@ -53,14 +53,21 @@ export function useLoadDataStreams() {
   });
 }
 
-// TODO: Implement this API endpoint once we have content to surface in the detail panel.
 export function useLoadDataStream(name: string) {
-  return useRequest<DataStream[]>({
-    path: `${API_BASE_PATH}/data_stream/${encodeURIComponent(name)}`,
+  return useRequest<DataStream>({
+    path: `${API_BASE_PATH}/data_streams/${encodeURIComponent(name)}`,
```

Comment:
I see the type annotation was changed to not being an array but the endpoint is pluralised, is this intended to be the case?

Reply:
Yes, this was a deliberate change to align with our other Index Management APIs, which consistently use plural nouns regardless of whether you're fetching all or one of a given entity.